src/test-fixtures/docmap-parser.ts

```diff
@@ -548,6 +550,56 @@ export const fixtures = {
     return generateDocMap('test', publisher, firstStep);
   },
 
+  inferredVersionOfRecordCorrected: (): DocMap => {
```

Comment:
Nitpick: This isn't inferred because the assertion is being explicit. Maybe `assertedVersionOfRecordCorrected` ?